src/http-gateways/path-gateway.md

```diff
@@ -641,9 +677,15 @@ Optional, present in certain response types:
 
 ### `Retry-After` (response header)
 
-Gateway returns this header with error responses such as [`429 Too Many Requests`](#429-too-many-requests) or [`504 Gateway Timeout`](#504-gateway-timeout).
+Gateway SHOULD return this header with error responses such as [`429 Too Many Requests`](#429-too-many-requests), [`504 Gateway Timeout`](#504-gateway-timeout) or `503` (server maintainance).
 
-The "Retry-After" header indicates how long the user agent ought to wait before making a follow-up request.
+The "Retry-After" header indicates how long the user agent ought to wait before making a follow-up request. It uses the following syntax:
+
+```
+Retry-After: <http-date>
+// or
+Retry-After: <delay-seconds>
```

Comment:
:thought_balloon:   Date is virtually never used in real world, most services (Nginx, AWS, Cloudflare, Caddy) default to using delay in seconds for simplicity and practicality. 

Putting it here introduces unnecessary complexity (similar to range requests that are for more than one range – they are in spec, but nothing implements them correctly enough for it to be used). 

Let's simplify

```suggestion
Retry-After: <delay-seconds>
```